From: Markus Koschany <apo@gambaru.de>
Date: Sat, 14 Sep 2013 22:41:26 +0200
Subject: CMakeList

---
 src/CMakeLists.txt | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 7047df1..8ccd378 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -122,13 +122,12 @@ if(UNIX)
 		DESTINATION
 			${CMAKE_INSTALL_PREFIX}/share/applications
 	)
-	install(
-		FILES
-			${MARS_SOURCE_DIR}/resources/mars
-		DESTINATION
-			${CMAKE_INSTALL_PREFIX}/share/menu
-	)
-
+    #install(
+    #FILES
+    #${MARS_SOURCE_DIR}/resources/mars
+    #DESTINATION
+    #${CMAKE_INSTALL_PREFIX}/share/menu
+    #)
 else(UNIX)
 	# executable
 	install(
